book/modules/appendix3.tex

```diff
@@ -0,0 +1,241 @@
+Determinant is useful: it measures the volume distortion of a linear transformation,
+it helps us determine if a matrix is invertible of not, and it provides us with a
+simple formula for solving $A\vec x=\vec y$ when $A$ is invertible.
+
+Module \ref{determinant module} discusses an algorithm for evaluating determinants
+via row reduction. However, for $2\times 2$ and $3\times 3$ determinants, we
+have direct formulas. We will explore these formulas in this appendix.
+
+\Heading{A formula for $2\times 2$ determinants} For a $2\times 2$ matrix, once we
```

Comment:
I think there should be some consistency among the headings. Something like `Computing 2x2 Determinants`, `Computing 3x3 Determinants (I)...`, `Computing 3x3 Determinants (II)...`, etc.